ethexe/service/src/tests.rs

```diff
@@ -155,22 +155,28 @@ async fn ping() {
         .expect("After approval, instrumented code is guaranteed to be in the database");
 
     let res = env
-        .create_program(code_id, b"PING", 0)
+        .create_program(code_id)
         .await
         .unwrap()
         .wait_for()
         .await
         .unwrap();
     assert_eq!(res.code_id, code_id);
-    assert_eq!(res.init_message_source, env.sender_id);
-    assert_eq!(res.init_message_payload, b"PING");
-    assert_eq!(res.init_message_value, 0);
-    assert_eq!(res.reply_payload, b"PONG");
-    assert_eq!(res.reply_value, 0);
+
+    let res = env
+        .send_message(res.program_id, b"PING", 0)
+        .await
+        .unwrap()
+        .wait_for()
+        .await
+        .unwrap();
+
     assert_eq!(
         res.reply_code,
         ReplyCode::Success(SuccessReplyReason::Manual)
     );
+    assert_eq!(res.reply_payload, b"PONG");
```

Comment:
overall refactor would be good as well 
```suggestion
    assert_eq!(reply.payload, b"PONG"); // etc
```